Gemfile.lock

```diff
@@ -0,0 +1,46 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    codecov (0.1.14)
+      json
+      simplecov
+      url
+    diff-lcs (1.3)
+    docile (1.3.1)
+    json (2.1.0)
+    rack (2.0.6)
+    rake (12.3.2)
+    robocop (0.1.1)
+      rack
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
+      rspec-support (~> 3.8.0)
+    rspec-expectations (3.8.2)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.8.0)
+    rspec-mocks (3.8.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.8.0)
+    rspec-support (3.8.0)
+    simplecov (0.16.1)
+      docile (~> 1.1)
+      json (>= 1.8, < 3)
+      simplecov-html (~> 0.10.0)
+    simplecov-html (0.10.2)
+    url (0.3.2)
+
+PLATFORMS
+  x64-mingw32
```

Comment:
I believe this line is causing install failures within the travis-ci build.

`mingw` is for windows only. You should probably just remove it.

Further reading notes we should not check in a `Gemfile.lock` file for gem development.

nice article -> https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Reply:
Got it. I'll just remove that file.